js/script.js

```diff
@@ -0,0 +1,14 @@
+function drawTree(a) {
+
+    for (var i = 1; i <= a; i++) {
```

Comment:
try to write it using only 2 loops

Reply:
Hmm... I need some clue. I can't figure out how to do this.

Reply:
i sent you solution on chat ;) add it to repo :)